### ISSUE TITLE
is this project still maintained?

### ISSUE THREAD

**Issue by iambumblehead** [APIs-guru/graphql-voyager#187]
is this project still maintained? are there alternatives or forks available?

**Comment by LunaticMuch** [comment]
I believe no. Which is a shame.

**Comment by iambumblehead** [comment]
@LunaticMuch it is shame. This voyager api tool is really one of the best web tools I've seen or used.

**Comment by IvanGoncharov** [comment]
@iambumblehead @LunaticMuch Thanks for the kind words about our project 😊 
We have big plans for moving this project forward but it requires a lot of work hours together with day-to-day maintenance. 
So we are in the process of finalizing a solution that will give us the necessary financing.
I will keep this issue open and will update you here.

**Comment by LunaticMuch** [comment]
Feel free to get to get in touch for any help.

**Comment by iambumblehead** [comment]
@IvanGoncharov that is great news! thank you for your response :)

**Comment by ifnull** [comment]
@IvanGoncharov I've been using this on a project over the last year and love it. Any chance you could create a new release with what is in the `master` branch? I see a lot of improvements on the live demo from what I see in 1.0.0-rc.31. Thanks!

**Comment by alfaproject** [comment]
@IvanGoncharov it's been a while and I'm wondering if you can give another update

**Comment by macrozone** [comment]
> Update: APIs.guru (as a consulting company) died years ago. My attempts to find financing by merging APIs.guru projects into another company failed. I discussed this situation with Roman (co-author) and he gave me full rights to this project.
> 
> Currently, I work at Apollo full-time but I have some spare time on weekends. So I will try to contribute and accept external contributions regularly. I moved all projects related to GraphQL to my personal account as a first step. Next weekend I plan to do basic cleanup on projects and update dependencies.

hi,

wouldn't this be a nice addition to apollo studio anyway? As far as I am aware, there is no visual representation of the graph in apollo studio:

[
<img width="612" alt="Bildschirmfoto 2022-07-21 um 16 08 12" src="https://user-images.githubusercontent.com/1972353/180234408-f89a8681-d15c-48e7-b96f-8a1b4919bb1f.png">
](url)


despite the fact that the icon indicates a graphical representation 😉:

<img width="128" alt="Bildschirmfoto 2022-07-21 um 16 08 01" src="https://user-images.githubusercontent.com/1972353/180234519-4b46516c-2e0a-494a-a7bf-e65eec33fdeb.png">


maybe you could pitch that on the team...?

**Comment by LunaticMuch** [comment]
@macrozone correct, there's not but this should be asked to Apollo directly. Not sure they might want.

**Comment by macrozone** [comment]
> @macrozone correct, there's not but this should be asked to Apollo directly. Not sure they might want.

(i mentioned it because @IvanGoncharov works at apollo 🤓 )

**Comment by LunaticMuch** [comment]
> (i mentioned it because @IvanGoncharov works at apollo 🤓 )

I know, that's why it should be better asked from outside 😉

**Comment by macrozone** [comment]
> > (i mentioned it because @IvanGoncharov works at apollo 🤓 )
> 
> I know, that's why it should be better asked from outside 😉

true, i opened a FR: https://github.com/apollographql/apollo-feature-requests/issues/337

**Comment by IvanGoncharov** [comment]
Recently, Voyager saw quite a big push with the `1.0.0` release 🎉 
So I want to use this opportunity and Voyager users to help my country to fight this war 🇺🇦 
I set up OpenCollective: https://opencollective.com/graphql-voyager
All collected money after expenses goes to Ukrainian paramedics deployed alongside troops.

cc: @iambumblehead @LunaticMuch @ifnull @alfaproject @dp28 @macrozone
